src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryMultiSinkConfig.java

```diff
@@ -42,6 +43,10 @@
     "arguments will be processed. If enabled, all records will be written as-is.")
   private Boolean allowFlexibleSchema;
 
+  public BigQueryMultiSinkConfig() {
```

Comment:
Can we also replace it with parameterized constructor??

Reply:
umm so do we want to do that to be consistent with other config files?

Reply:
Yeah, also I would like to keep the constructor private and call it from builder.

Reply:
got it!